qt/aqt/webview.py

```diff
@@ -968,3 +961,53 @@ def on_body_classes_need_update(self) -> None:
     @deprecated(info="use theme_manager.qcolor() instead")
     def get_window_bg_color(self, night_mode: bool | None = None) -> QColor:
         return theme_manager.qcolor(colors.CANVAS)
+
+
+# Pre-configured classes for use in Qt Designer
```

Comment:
Some of this might be YAGNI and over-engineered for the current use case, but given that `AnkiWebView` is the custom widget we use the most in Qt forms, I wanted to set things up in a way where - if we ever saw a need to supply other (new) construction time arguments - we could do so very easily.